java/runtime/src/main/java/org/ray/runtime/context/NativeWorkerContext.java

```diff
@@ -38,6 +38,11 @@ public ActorId getCurrentActorId() {
     return ActorId.fromByteBuffer(nativeGetCurrentActorId(nativeCoreWorkerPointer));
   }
 
+  @Override
+  public boolean getIsDirectCall() {
```

Comment:
It seems that we don't need to expose this method in Java. Because this var will only be passed into c++ `PrepareActorCheckpoint`. So we directly get this variable in C++.